example/views/index.jsx

```diff
@@ -1,10 +1,11 @@
 const React = require('react');
 const Layout = require('./layout');
-
 const Input = require('../../components/forms/input-text');
 const RadioGroup = require('../../components/forms/radio-group');
 const Select = require('../../components/forms/select');
 const Date = require('../../components/forms/date');
+const SubNavBar = require('../../components/subnavbar').default;
```

Comment:
It's _really_ nasty IMO that this needs a `.default` :-(

Reply:
I agree.. Is there a better way to expose multiple exports?

Basically want the 'default' to be the component and then additional items available as well.

Below is how this looks on a 'create-react-app' project. 

```jsx
import SubNavBar from '../../components/subnavbar';

// or .. 
import { SubNavBar, NavItem } from '../../components/subnavbar
```

Reply:
What I meant to say before I left the office an got distracted, was that I think it's possible to use ES modules in the views, and the react view engine will compile them on the fly. That way jsx just always uses `import` and it "just works".

I'll give it a try.

Reply:
^^ OK awesome.. That'd be good!